docs/accounts.md

```diff
@@ -0,0 +1,21 @@
+# Blacklight Login Scenarios 
+
+## Unauthenticated User from main menu
+
+## Library Account - From the User Menu Bar
+    * with Netid - devise user type "cas"
+    * with your Alma active barcode -credentials are Last Name and active barcode  devise user type "barcode" - this is a legacy option, no new accounts being set up in this fashion. 
+    * with your Alma internal account credentials (Username is the "primary identifer" field in user's Alma account, password is set and stored in Alma) - devise user type "alma" 
+
+
+## Bookmarks and Search History - Blacklight User Account https://catalog.princeton.edu/bookmarks and  (Login option is buried in a link)
+    a. with Netid - devise user type "cas"
+    b. with your Alma internal account credentials (Username is the "primary identifer" field in user's Alma account, password is set and stored in Alma) - devise user type "alma" 
+
+
+## Unauthenticated User from a request page
+    * with Netid - devise user type "cas"
```

Comment:
same change here . If we could add what exactly is in the UI and then the explanation that you provide.